deepchem/models/tests/test_openvino.py

```diff
@@ -4,10 +4,53 @@
 import torch
 
 import deepchem as dc
-from deepchem.molnet import load_tox21
+from deepchem.molnet import load_tox21, load_perovskite, load_bace_classification
 from deepchem.data import NumpyDataset
 from deepchem.models.torch_models import TorchModel
 
+from os import path, remove
+
+def test_cgcnn():
+  # load datasets
+  current_dir = path.dirname(path.abspath(__file__))
+  config = {
+      "reload": False,
+      "featurizer": dc.feat.CGCNNFeaturizer(),
+      # disable transformer
+      "transformers": [],
+      "data_dir": current_dir
+  }
+ 
+  tasks, datasets, transformers = load_perovskite(**config)
+  train, valid, _ = datasets
+
+  n_tasks = len(tasks)
+
+  def init(**kwargs):
+#     tf.random.set_seed(572)
```

Comment:
Why `tf` (TensorFlow)? We need to initialize PyTorch model with the same random weights every time.